http/vibe/http/auth/basic_auth_client.d

```diff
@@ -0,0 +1,21 @@
+/**
+	Implements HTTP Basic Auth for client.
+
+	License: Subject to the terms of the MIT license, as written in the included LICENSE.txt file.
+*/
+module vibe.http.auth.basic_auth_client;
```

Comment:
Why the new module ? If it's only for one function I am not sure it's worth it.

Reply:
It is need to avoid cyclic dependency of ctors/dtors between HTTP client and server modules what both use `addBasicAuth`